membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -230,13 +230,18 @@ class AttributeController(
   def oneOffContributions =
     AuthorizeForScopes(requiredScopes = List(readSelf)).async { implicit request =>
       metrics.measureDuration("GET /user-attributes/me/one-off-contributions") {
+        import request.logPrefix
         val userHasValidatedEmail = request.user.userEmailValidated.getOrElse(false)
 
         val futureResult: Future[Result] =
           if (userHasValidatedEmail) {
             contributionsStoreDatabaseService.getAllContributions(request.user.identityId).map {
-              case Left(err) => Ok(err)
```

Comment:
I don't think we should have been returning the postgres error to the client, there doesn't seem to be any discussion on the original pr to say it's needed https://github.com/guardian/members-data-api/pull/388/files#r1969905410

Reply:
good spot!